src/main/java/com/ldtteam/blockout/Pane.java

```diff
@@ -805,4 +831,14 @@ public boolean onMouseDrag(final double mx, final double my, final int speed, fi
     {
         return false;
     }
+
+    /**
+     * Sets the tooltip to render on hovering this element
+     *
+     * @param lines the lines to display
+     */
+    public void setHoverToolTip(final List<String> lines)
```

Comment:
There should probably be a way to set it via pane params too.

Reply:
nah this is easier :D

Reply:
Agree with Ray honestly

Reply:
Oh, I mean we should have both in this regards.

Reply:
I think we only should pass essential and required information as constructor parameters, not optional ones. Else we run into a pleathora of default constructors with less parameters for each and every class extending pane

Reply:
not in the constructor. To the paneparams object.

Reply:
should've said so to start with! lately you're very sparse with details xD

Reply:
Marie Kondo